src/movies.js

```diff
@@ -1,16 +1,79 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(array){
+    let directorsArr = array.map((elem)=>{
+    return elem.director
+    })
+    return directorsArr
+    }
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-
+function howManyMovies(array){
+    
+    let filterArray = array.filter((elem)=>{
+        return (elem.director==="Steven Spielberg" && elem.genre.indexOf("Drama") != -1)
```

Comment:
Good job on putting the two conditions on the same line. indexOf works good here, however this method is mostly used when you actually want to know the index in case it exists. To know only if an element exists or not inside an array you can also use .includes which returns a boolean.